project/Play2-ReactiveMongo.scala

```diff
@@ -122,7 +130,8 @@ object Play2ReactiveMongoBuild extends Build {
         "Sonatype snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/",
         "Sonatype" at "http://oss.sonatype.org/content/groups/public/",
         "Typesafe repository releases" at "http://repo.typesafe.com/typesafe/releases/",
-        "Typesafe repository snapshots" at "http://repo.typesafe.com/typesafe/snapshots/"
+        "Typesafe repository snapshots" at "http://repo.typesafe.com/typesafe/snapshots/",
+        "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases"
```

Comment:
Yes, it is. :-(

Quoting [Play 2.4 migration guide](https://www.playframework.com/documentation/2.4.x/Migration24#Specs2-support-in-a-separate-module):

> Additionally, specs2 now requires scalaz-stream which isn’t available on maven central or any other repositories that sbt uses by default.